guide/onboarding/automatic-cloud-backup.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Header image is missing on this page

Reply:
I don't think it ever had an image, but I can add one in.
  
I'm almost wondering if we should tweak header images for all the Onboarding pages since we are trying to move away from this idea that all the onboarding needs to happen in a specific order.

Reply:
I added in the one from the main Backups > Onboarding page for now.